frontend/src/app/lib/inject.js

```diff
@@ -1,34 +1,20 @@
 import React from "react";
-import ReactDOM from "react-dom";
 import { Provider } from "react-redux";
-
-import { setupAddonConnection } from "./InstallManager";
-import store from "../store";
+import { StaticRouter } from "react-router";
 import App from "../containers/App";
+import store from "../store";
 
 export default function inject(name, component, callback) {
```

Comment:
This is no longer injecting the node into the dom, so it's kind of badly named now.

Reply:
Yeah, I think we should do another pass of refactoring around this. I wanted to make this diff about as small as I could to start.

Reply:
It might be better just to copy paste the implementation of this into every place that calls inject now. `name` seems to no longer be used and callback is now also no longer used.